Wrapping/TypedefMacros.cmake

```diff
@@ -316,13 +316,8 @@ macro(itk_wrap_named_class class swig_name)
 
   if("${ARGC}" EQUAL 3)
     set(WRAPPER_WRAP_METHOD "${ARGV2}")
-    set(ok 0)
-    foreach(opt POINTER POINTER_WITH_CONST_POINTER POINTER_WITH_SUPERCLASS POINTER_WITH_2_SUPERCLASSES EXPLICIT_SPECIALIZATION POINTER_WITH_EXPLICIT_SPECIALIZATION ENUM AUTOPOINTER)
-      if("${opt}" STREQUAL "${WRAPPER_WRAP_METHOD}")
-        set(ok 1)
-      endif()
-    endforeach()
-    if(ok EQUAL 0)
+    set(VALID_WRAP_METHODS POINTER POINTER_WITH_CONST_POINTER POINTER_WITH_SUPERCLASS POINTER_WITH_2_SUPERCLASSES EXPLICIT_SPECIALIZATION POINTER_WITH_EXPLICIT_SPECIALIZATION ENUM AUTOPOINTER)
+    if( NOT "${WRAPPER_WRAP_METHOD}" IN_LIST VALID_WRAP_METHODS)
       message(SEND_ERROR "itk_wrap_class: Invalid option '${WRAPPER_WRAP_METHOD}'. Possible values are POINTER, POINTER_WITH_CONST_POINTER, POINTER_WITH_SUPERCLASS, POINTER_WITH_2_SUPERCLASSES, EXPLICIT_SPECIALIZATION, POINTER_WITH_EXPLICIT_SPECIALIZATION, ENUM and AUTOPOINTER")
```

Comment:
Possibly reduce duplication by using "Possible values (semicolon separated): ${VALID_WRAP_METHODS}"

Reply:
Sorry, I don't understand this comment.  The "IN_LIST" item requires a variable on the right-hand side, not a list.

Reply:
I suggest we print the content of that variable here, to prevent this message getting out of sync with the list which is actually checked.

Reply:
I mean output the content of that variable in this message.